apps/api/src/routers/admin.py

```diff
@@ -172,6 +172,47 @@ async def submit_review(
         raise HTTPException(status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+@router.post("/review/hacker")
+async def submit_hacker_review(
+    applicant: str = Body(),
+    score: float = Body(),
+    reviewer: User = Depends(require_role({Role.REVIEWER})),
+) -> None:
+    """Submit a review decision from the reviewer for the given hacker applicant."""
```

Comment:
Thought: do you know what the review process for the mentors/volunteers will look like? Maybe instead of creating separate endpoints, we could combine them and use dynamic logic based on the applicant type inside for the reviewer count. If the plan for mentors/volunteers is to use discrete discussions, these could be internally encoded as numeric scores (e.g. accept = 100, reject = 0) while the UI shows the decision names.

Reply:
After discussion with @waalbert , the review process for mentors/volunteers should be the exact same as last year. So 1 reviewer with the same review logic as before. The only thing that would change is separating them into their own tabs in the portal. This would keep us from having to rewrite all of the review logic.

Reply:
Yep, in this case we can have all apps share the same endpoint and based on the type of app, assign the scores accordingly.